Sources/ProjectDescription/Target.swift

```diff
@@ -17,8 +17,8 @@ public struct Target: Codable, Equatable {
     /// The product bundle identifier.
     public let bundleId: String
 
-    /// The minimum deployment target your product will support.
-    public let deploymentTarget: DeploymentTarget?
```

Comment:
What does it mean to have a single platform but multiple deployment targets? shouldn't we remove the platform parameter?